src/components/Cluster/ClusterDetail/ClusterDetailView.js

```diff
@@ -114,7 +113,6 @@ class ClusterDetailView extends React.Component {
     this.props.dispatch(
       batchedRefreshClusterDetailView(clusterID, this.props.isV5Cluster)
     );
-    this.props.dispatch(loadApps(clusterID));
```

Comment:
Removed this and placed this disapatch in its corresponding batched function. 

The whole point of batched functions is to centralize series of API calls that a view needs. We decided to do this as a way of self-documenting the code, and also for debugability. 

If place dispatches consecutively in the view, as we had before, we are cheating ourselves.

Also, this was triggering an error for deleted clusters, because the thunk was dispatched no matter if the cluster existed or not.